DetectionFactory/ConvertDetection.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using Business.Detection.Common.Models;
+using Data;
+using Data.Detection;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace FactoryDetectionTests
+{
+    public class ConvertDetection
```

Comment:
By convention the name of the test class should be composed of: `[NameOfTheClassBeingTested][Tests]`. Example:

- `FooClass` - `FooClassTests`
- `RandomGenerator` - `RandomGeneratorTests`
- `RandomFactory` - `RandomFactoryTests`

With those said, should we consider renaming this test class to match the above convention?

Reply:
Done